lib/cog/repo.ex

```diff
@@ -1,3 +1,13 @@
 defmodule Cog.Repo do
   use Ecto.Repo, otp_app: :cog
+
+  alias Cog.Queries.Repo, as: Queries
+
+  def exists?(model, id, opts \\ [true: true, false: false]) do
```

Comment:
Ah, that's not necessary anymore. That's because I wanted to be able to optionally return `:found` or `:not_found` but the code that needed it got brutally refactored. I'll rip that out.